wdl/wdl-core/src/main/scala/wdl/versioning/WdlVersionSpecifics.scala

```diff
@@ -0,0 +1,10 @@
+package wdl.versioning
+
+import wom.values.{WomFloat, WomValue}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Try
+
+trait WdlVersionSpecifics {
+  def sizeFunctionOverride(params: Seq[Try[WomValue]], sizeFunc: WomValue => Try[Double]): Try[WomFloat]
```

Comment:
The trait which carries around all of the functions to slot into the various holes in `wdl-core`